### PR TITLE
Kibana all-in-one is initialized in the same way as the distributed installation

### PR DESCRIPTION
**Pull request by verdx**
|Related issue|
|---|
||

## Description

Added initialize option to kibana all-in-one so it waits for kibana to be up to continue the installation.

### CODE REVIEW

File: unattended_scripts/install_functions/opendistro/elasticsearch.sh [hunk 1]
```diff
@@ -182,7 +182,7 @@ configureElasticsearch() {
     eval "rm /etc/elasticsearch/certs/client-certificates.readme /etc/elasticsearch/certs/elasticsearch_elasticsearch_config_snippet.yml -f ${debug}"
     eval "/usr/share/elasticsearch/bin/elasticsearch-plugin remove opendistro-performance-analyzer ${debug}"
 
-    initializeElastic nip
+    initializeElastic
```
Comment (alberpilot) [inline unattended_scripts/install_functions/opendistro/elasticsearch.sh:185]: `Elastic` is the company name. The component is `Elasticsearch`. Please change all occurrences.

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

**Review by alberpilot — APPROVED** [review]
LGTM

**Review by rauldpm — APPROVED** [review]
LGTM

**Pull request merged**